Add how to integrate with COS documentation (#233)
canonical/wazuh-server-operator: docs/how-to/integrate-with-cos.md

Reviewer: I think you'll need to add this to the `Contents` section of the `docs/index.md` page

> # Integrate with Canonical Observability Stack

Reviewer: ```suggestion
# How to integrate with Canonical Observability Stack
```

```

## Expose the application relation endpoints
Once all the COS Lite applications are deployed and settled down (you can monitor this by using `juju status --watch 2s`), expose the relation points for Prometheus, Loki and Grafana:

```bash
juju offer prometheus:metrics-endpoint
juju offer loki:logging
juju offer grafana:grafana-dashboard
```

Validate that the offers have been successfully created by running:

```bash
juju find-offers cos-lite
```

You should see something similar to the output below:

```bash
Store                 URL                        Access  Interfaces
tutorial-controller  admin/cos-lite.loki        admin   loki_push_api:logging
tutorial-controller  admin/cos-lite.prometheus  admin   prometheus_scrape:metrics-endpoint
tutorial-controller  admin/cos-lite.grafana     admin   grafana_dashboard:grafana-dashboard
```

## Integrate the Wazuh server

Switch back to the charm model and integrate your charm with the exposed endpoints:

```bash
juju switch <Wazuh server charm model>
juju integrate wazuh-server admin/cos-lite.grafana
juju integrate wazuh-server admin/cos-lite.loki
juju integrate wazuh-server admin/cos-lite.prometheus
```

> Ensure that Hockeypuck and the COS Lite applications are settled down (you can monitor this by using `juju status --watch 2s`).

Reviewer 2: ```suggestion
Ensure that Wazuh server and the COS Lite applications are settled down (you can monitor this by using `juju status --watch 2s`).
```

> Ensure that Hockeypuck and the COS Lite applications are settled down (you can monitor this by using `juju status --watch 2s`).

Reviewer 3: Hockeypuck ;)

> Ensure that Wazuh server and the COS Lite applications are settled down (you can monitor this by using `juju status --watch 2s`).

Reviewer: ```suggestion
Ensure that Wazuh server and the COS Lite applications have settled down (you can monitor this by using `juju status --watch 2s`).
```
Nitpick

Obtain the Grafana dashboard credentials by running the `get-admin-password` action:

```bash
juju switch cos-lite
juju run grafana/0 get-admin-password
```

> This action returns the URL and the admin password to access the Hockeypuck dashboard. Now, on your host machine, open a web browser, enter the Grafana URL, and use the username “admin” and your Grafana password to log in. Under **Home > Dashboards**, you should be able to see the Hockeypuck dashboard listed.

Reviewer 3: Twice.

> This effectively integrates your application with Prometheus, Loki, and Grafana.

Reviewer: What do you mean by "effectively"? Is there a different way a user could go about integrating Wazuh with COS?
